gson/pom.xml

```diff
@@ -1,67 +1,74 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Why are all the gson files in this PR? Are these leftover from another article/PR?